sandbox/sandbox.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
@mavenugo Sure, I can rework this if you feel necessary (I tried and failed on 2 attempts - it's really not that easy).  But, I really do have to ask why. You're asking to keep an interface in a platform which isn't being used, requiring far more code to be pulled in for compilation and execution path than is necessary, and making x-platform CI harder for everyone. That IMO simply doesn't make sense. Not meaning to vent - trying to be constructive, please take it that way :)